### PR TITLE
Release 0.1.12

### PR DESCRIPTION
**Pull request by fernandog**

### CODE REVIEW

**Comment by fernandog** [comment]
@OmgImAlexis why is failling?

**Comment by OmgImAlexis** [comment]
You're using another branch, this is setup to use master. You're meant to just push to master. I've said this multiple times.

**Comment by OmgImAlexis** [comment]
Just merge and it should fix itself up. The idea here is that we don't want anyone to just open a PR and have that replace the live site. 

Maybe we should change the command to use check if the branch is master and only try and push when it is?

**Comment by fernandog** [comment]
@OmgImAlexis

Cant merge as we didn't release yet.
Need to create  PR

File: static/news/news.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
-####Please read the [Wiki/FAQ](https://github.com/pymedusa/SickRage/wiki) before opening an issue####
-All issues and bug reports must be opened at [GitHub](https://github.com/pymedusa/SickRage/issues)
+####Please read the [Wiki/FAQ](https://github.com/pymedusa/Medusa/wiki) before opening an issue####
+All issues and bug reports must be opened at [GitHub](https://github.com/pymedusa/Medusa/issues)
+
+####2017-04-21####
+
+## 0.1.12
+
+This release contains many bug fixes and a few useful new features. First of all, we would like to say thank you to all users who took the time to report bugs. We take your feedback very seriously and we constantly try to improve Medusa.
```
Comment (ratoaq2) [inline static/news/news.md]: Trying to improve

File: static/news/news.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+
+Let's talk about the new features now. A lot of users have reported slow loading times with huge libraries, this should be much better now with the lazy-loading of images. Your backlog has always been too messy? We made some substantial changes that give you more control over the overview, making it easier and more straightforward to manage your pending shows. An exclusive (for now) feature for Transamission and Deluge users has been added as well. Completed downloads can be moved to a seeding folder of choice, making it easier to handle files that are waiting to be processed. Make sure to check it out if interested. Now you can also update scene exceptions or sync Trakt shows manually if required.
```
Comment (ratoaq2) [inline static/news/news.md]: Transmission

**Pull request merged**